app-text/ghostscript_gpl/patches/ghostscript_gpl-10.01.1.patchset

```diff
@@ -90,66 +67,66 @@ index 3312477..f627cbf 100644
  
  large_color_index=1
 -- 
-2.30.2
+2.37.3
 
 
-From 77e1d546ecdd50d42681611cfe614db60aa6c5ff Mon Sep 17 00:00:00 2001
+From 37e790a9a009daead7ff98968bc673881175df8c Mon Sep 17 00:00:00 2001
 From: Anarchos <sylvain_kerjean@hotmail.com>
 Date: Wed, 2 Mar 2022 21:15:32 +0100
 Subject: debug flags for haiku
 
 
 diff --git a/configure.ac b/configure.ac
-index f627cbf..d6563de 100644
+index cbe25a1..6f0e98c 100644
 --- a/configure.ac
 +++ b/configure.ac
-@@ -284,7 +284,7 @@ SET_DT_SONAME="-soname="
+@@ -280,7 +280,7 @@ SET_DT_SONAME="-soname="
  CFLAGS_LARGEFILE=""
  
  case $host in
 -        *-linux*|*-gnu)
 +        *-linux*|*-gnu|*-haiku)
-         if test $GCC = yes; then
+         if test x"$GCC" = x"yes"; then
              CC_OPT_FLAGS_TO_TRY="-O2 -DNDEBUG"
              CC_DBG_FLAGS_TO_TRY="-gdwarf-2 -g3 -O0"
 -- 
-2.30.2
+2.37.3
 
 
-From 41b392e739aa16d943788b3a889d8a438efd1459 Mon Sep 17 00:00:00 2001
+From 9ca5737d3dd79aba9c171121e229c3c601861c43 Mon Sep 17 00:00:00 2001
 From: Anarchos <sylvain_kerjean@hotmail.com>
 Date: Fri, 4 Mar 2022 05:47:01 +0100
 Subject: -g
 
 
 diff --git a/configure.ac b/configure.ac
-index d6563de..d682a97 100644
+index 6f0e98c..68d42e5 100644
 --- a/configure.ac
 +++ b/configure.ac
-@@ -287,7 +287,7 @@ case $host in
+@@ -283,7 +283,7 @@ case $host in
          *-linux*|*-gnu|*-haiku)
-         if test $GCC = yes; then
+         if test x"$GCC" = x"yes"; then
              CC_OPT_FLAGS_TO_TRY="-O2 -DNDEBUG"
 -            CC_DBG_FLAGS_TO_TRY="-gdwarf-2 -g3 -O0"
 +            CC_DBG_FLAGS_TO_TRY="-g"
              CC_VISIBILITY_FLAGS_TO_TRY="-fvisibility=hidden"
          fi
          ;;
 -- 
-2.30.2
+2.37.3
 
 
-From 8fd422a85278e54141866273a61d3eeb0d2245b1 Mon Sep 17 00:00:00 2001
+From 978e4572d5186fc354aa99258cdeba29f3682ebc Mon Sep 17 00:00:00 2001
```

Comment:
This could be removed together with the next commit. The next one below reverts this one.